.bumpversion.cfg

```
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.0.0
+current_version = 
```

Fix bumpversion bug.

```suggestion
current_version = 2.0.1
```